test/test_bulk_actions.py

```diff
@@ -219,6 +230,7 @@ def test_clear_incident_in_metadata(metadata):
     assert endpoints["link"][0]["events"][0]["clear_time"] == clear_time
     assert endpoints["link"][0]["events"][0]["properties"]["oper_up_time"] == clear_time
     assert endpoints["link"][0]["events"][0]["properties"]["oper_up_time"] == clear_time
+    assert is_naive_isoformat(endpoints["link"][0]["events"][0]["clear_time"])
```

Comment:
This additional assert doesn't add any value, it just checks your test setup code, rather than production code. best to remove it